packages/contracts/test/zrx_token.ts

```diff
@@ -21,12 +21,6 @@ describe('ZRXToken', () => {
     let MAX_UINT: BigNumber;
     let zrxToken: ZRXTokenContract;
 
-    before(async () => {
-        await blockchainLifecycle.startAsync();
-    });
-    after(async () => {
-        await blockchainLifecycle.revertAsync();
-    });
```

Comment:
I think these are unsucessful merges. We want these in here. Ditto in `unlimited_allowance_token.ts` and `token_registry.ts`.